extension/src/openvic-extension/GameSingleton.cpp

```diff
@@ -10,86 +10,113 @@
 using namespace godot;
 using namespace OpenVic;
 
+using OpenVic::Utilities::godot_to_std_string;
+using OpenVic::Utilities::std_to_godot_string;
+using OpenVic::Utilities::std_view_to_godot_string;
+
 GameSingleton* GameSingleton::singleton = nullptr;
 
+#define BM ClassDB::bind_method
+#define BSM ClassDB::bind_static_method
```

Comment:
Probably could've done
```suggestion
#define BM_SIMPLE  ClassDB::bind_method
#define BM(MethodName, ...) BM_SIMPLE(D_METHOD(#MethodName __VA_OPT__(,) __VA_ARGS__), &GameSingleton::MethodName)
#define BSM_SIMPLE ClassDB::bind_static_method
#define BSM(MethodName, ...) BSM_SIMPLE("GameSingleton", D_METHOD(#MethodName __VA_OPT__(,) __VA_ARGS__), &GameSingleton::MethodName)
```
if we're gonna do macros here, but it doesn't really matter.

Reply:
I added the macros to shorten line lengths, as almost every call to those functions was just too long to fit on a single line. For now I just want to get this PR merged without any further delay, but your improved macros are definitely worth adding in a future PR.